src/test/java/GmailActionableEmailsServletTest.java

```diff
@@ -150,21 +190,45 @@ public void nDaysZero() throws Exception {
   }
 
   @Test
-  public void validResponse() throws Exception {
+  public void validResponseSortByDate() throws Exception {
     Mockito.when(request.getParameter("subjectLinePhrases"))
         .thenReturn(SUBJECT_LINE_PHRASES_STRING);
     Mockito.when(request.getParameter("unreadOnly")).thenReturn(String.valueOf(true));
     Mockito.when(request.getParameter("nDays")).thenReturn(String.valueOf(DEFAULT_N_DAYS));
+    Mockito.when(
+            actionableMessageHelper.assignMessagePriority(Mockito.any(), Mockito.eq(USER_EMAIL)))
```

Comment:
Is it possible to narrow down `Mockito.any()` any further?

Reply:
Maybe, but I almost want to keep this for clarity. This unit test does not consider priority - I want to make it clear that any passed in message will just have the default priority, so sorting will be only based on the secondary factor, `internalDate`

Reply:
@antmar @gbuenoandrade let me know your thoughts on this or if you have any suggestions!

Reply:
sgtm

Reply:
RESOLVED